Modules/Core/Common/test/itkMersenneTwisterRandomVariateGeneratorGTest.cxx

```diff
@@ -64,14 +64,12 @@ TEST(MersenneTwisterRandomVariateGenerator, GetIntegerVariateReturnsSameAsStdMt1
 }
 
 
+// Tests that two GetNextSeed() calls return the same very seed, when ResetNextSeed() is called beforehand.
```

Comment:
the same very seed : the very same seed

Reply:
Good catch, Jon! Fixed by [this force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/838e9be4aa8dd904258bf83680d56f49712dea0f..d871a24a825cfe6d60180f4ba6f1d8fa42fe3a9f.)